scripts/rpc.js

```diff
@@ -0,0 +1,12 @@
+function RPC(title) {
```

Comment:
Could you rename it to something that contains the name Discord? E.g: DiscordRPC
for clarity

Reply:
Sure I can change that right now.